configure.ac

```diff
@@ -638,3 +659,8 @@ if test "x$build_dtls" = "xyes"; then
 else
 	AC_MSG_RESULT([      build with DTLS support : "no"])
 fi
+
+#
+# Bring .map / .sym files up to date
+#
+make update-map-file > /dev/null 2>&1
```

Comment:
.PHONY does not cause them to get automatically updated / built, it just means that whenever 'make update-map-file' is called, it will always get run, even if there is a file called update-map-file.

When a built library is distributed, the distribution does not need to include the .map/.sym files - they are only used when creating the the library files.

As you are in favor of keeping / maintaining the .map/.sym files, I will revert out that change and re-install the distclean warning.

Reply:
Yes, correct.